src/web/client/common/remoteServiceProvider.ts

```diff
@@ -0,0 +1,147 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+'use strict';
```

Comment:
we don't need to add this explicitly, since it's there in the tsconfig compiler options.